src/GridList/GridTile.js

```diff
@@ -111,6 +112,10 @@ class GridTile extends Component {
      */
     titleBackground: PropTypes.string,
     /**
+     * Style used for title color.
+     */
+    titleColor: PropTypes.string,
```

Comment:
I agree! I was about to submit an issue out of a need to change the title's fontsize.